### PR TITLE
Volume material mapping

### PR DESCRIPTION
**Pull request by Corentin-Allaire**

### CODE REVIEW

**Comment by asalzburger** [comment]
@Corentin-Allaire - can you rebase and check if you can address some of the comments?

File: Core/include/Acts/Material/MaterialGridHelper.hpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+///////////////////////////////////////////////////////////////////
```
[the remark this answers is not in the document]
Comment (Corentin-Allaire) [reply Core/include/Acts/Material/MaterialGridHelper.hpp]: I removed them for the entire Material directory (and updated the Copyright years)

**Comment by asalzburger** [comment]
@Corentin-Allaire - could you rebase please?

**Comment by Corentin-Allaire** [comment]
Sorry, done.

**Comment by asalzburger** [comment]
I see this reduces the code covering by 2.5 % - is this because you can't really test this in unit test setups?

**Comment by Corentin-Allaire** [comment]
Some of it is definitely because it is hard to test the mapping without a proper geometry (I think it is the same with the surface mapping). Same for the json geometry writer that is hard to test without a proper geometry. I will still have a quite look a the rest of the files to see if I can do more in term of unittest.

**Comment by Corentin-Allaire** [comment]
@asalzburger when #82 is resolved do you think it will be possible to use the ODD or the Generic detector to test the surface/volume mapping and the json writer ? 
If not I could try to build a test geometry but that would probably be better to do in a separate issue that improve the coverage for the Material in general.

**Comment by Corentin-Allaire** [comment]
Also something weird I just notice, for Material.hpp I just changed the copyright year to 2019 to 2020 and the coverage goes down by 10 % ?

**Pull request merged**